### PR TITLE
add build status badges to README

### PR DESCRIPTION
**Pull request by swills**
Just a "nice to have" thing, if you're interested.

### CODE REVIEW

**Comment by alazymeme** [comment]
Is there a reason this can't be added along the top Chatterino heading? Why should there be another heading just for a build status?

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ You might also need to install the [VC++ 2017 Redistributable](https://aka.ms/vs
 If you still receive an error about `MSVCR120.dll missing`, then you should install the [VC++ 2013 Restributable](https://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x64.exe
 ).
 
+## Build Status
+[![GitHub Actions Build (Windows, Ubuntu, MacOS)](https://github.com/Chatterino/chatterino2/workflows/Build/badge.svg?branch=master)](https://github.com/Chatterino/chatterino2/actions?query=workflow%3ABuild+branch%3Amaster)
+[![Cirrus CI Build (FreeBSD only)](https://api.cirrus-ci.com/github/Chatterino/chatterino2.svg?branch=master)](https://cirrus-ci.com/github/Chatterino/chatterino2/master)
+
```
Comment (alazymeme) [inline README.md:20]: As aforementioned, these badges shouldn't have their own header. My suggestion would be to have them inline with the Chatterino 2 header. [Example](https://github.com/pajbot/pajbot)

**Review by alazymeme — CHANGES_REQUESTED** [review]

**Review by alazymeme — APPROVED** [review]
image

**Review by pajlada — APPROVED** [review]
:+1:

**Pull request merged**